documentation/images/plantuml/beerocks_cli/update_credentials.puml

```diff
@@ -0,0 +1,10 @@
+@startuml
+ -> beerocks_cli: beerocks_cli -c \n"bml_update_wifi_credentials al-mac ssid network_key radio bss_type"
+beerocks_cli -> beerocks_cli: cli_bml::update_wifi_credentials_caller(args)
+beerocks_cli -> beerocks_cli: cli_bml::update_wifi_credentials(\n args.stringArgs[0],\n args.stringArgs[1],\n args.stringArgs[2],\n args.stringArgs[3],\n args.stringArgs[4])
+beerocks_cli -> beerocks_cli: bml_update_wifi_credentials(\n ctx,\n al-mac.c_str(),\n ssid.c_str(),\n network_key.c_str(),\n radio.c_str(),\n bss_type.c_str())
+beerocks_cli -> bml_internal: update_wifi_credentials(\n const son::wireless_utils::sBssInfoConf &credentials,\n const sMacAddr &al_mac)
+bml_internal -> controller: message_com::send_cmdu(\n m_sockMaster,\n ACTION_BML_WIFI_CREDENTIALS_UPDATE_REQUEST)
+controller -> controller_db: update_bss_info_configuration(\n const sMacAddr &al_mac,\n const wireless_utils::sBssInfoConf &bss_info)

```

Comment:
This doesn't do anything in the controller db. Instead, it sends out an AUTOCONFIG_RENEW message over the network. So this should rather be something like:
```suggestion
controller -> controller: send_cmdu_to_bus(\n AP_AUTOCONFIGURATION_RENEW_MESSAGE,\n al_mac, db.get_local_bridge_mac());
```